packages/syft/src/syft/core/tensor/smpc/mpc_tensor.py

```diff
@@ -732,6 +744,10 @@ def __apply_op(
         shape = utils.get_shape(op_str, self.shape, y_shape)
         ring_size = utils.get_ring_size(self.ring_size, y_ring_size)
 
+        if utils.ispointer(result[0]):
```

Comment:
Does this code just get the result of the dict but assign it to nothing?

Reply:
@IonesioJunior was this supposed to set the boolean to True for these?

Reply:
Yup, you're right! Just fixed it now, thank you for reviewing and noticing it.

Reply:
Fixed by: [e47e21d](https://github.com/OpenMined/PySyft/pull/6918/commits/e47e21d3ed28b150676f4c5884b6677cf32a10c0)